Functions.Templates/Templates/SqlInputBinding-Typescript/index.ts

```diff
@@ -0,0 +1,24 @@
+import { AzureFunction, Context, HttpRequest } from "@azure/functions"
+
+/**
+ * Sample SQL Input Binding
+ * See https://aka.ms/sqlbindingsinput for more information about using this binding
+ *
+ * These tasks should be completed prior to running :
+ *      1. Update "commandText" in function.json - this should be the name of the table that you wish to upsert values to
```

Comment:
commandText should be the query command for input bindings right?

Reply:
Oh what the, thought I fixed that. Whoops.